<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrayProcessSettingDialog</class>
 <widget class="QWidget" name="GrayProcessSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QRadioButton" name="radioRed">
     <property name="text">
      <string>By RED component</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioGreen">
     <property name="text">
      <string>By GREEN component</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioBlue">
     <property name="text">
      <string>By BLUE component</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioAverage">
     <property name="text">
      <string>By component average </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioSmart">
     <property name="text">
      <string>By smart ( Red 29.9% | Green 58.7% | Blue 11.4% )</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
